frontend/public/components/utils/horizontal-nav.tsx

```diff
@@ -201,7 +203,21 @@ export const HorizontalNav: React.FC<HorizontalNavProps> = React.memo((props) =>
     (extraObjs, key) => ({ ...extraObjs, [key]: _.get(props[key], 'data') }),
     {},
   );
-  const pages = props.pages || props.pagesFor(_.get(props.obj, 'data'));
+  const navTabExtensions = _.get(props.obj, 'data')
```

Comment:
we should use optional chaining and avoid lodash `get` now.

Reply:
Right, we also did this in #3907 - it's a great way to eliminate those `_.get` calls.

```ts
const navTabExtensions = props.obj?.data
```